Software workshop blog post

karthik opened the pull request:
Please review and recommend changes. Original Google doc is here (has additional comments):  https://docs.google.com/document/d/1tGjdwyFuglsIk6MG18f1jbjDRULfwLERwboiANZRuOs/edit?usp=sharing

danielskatz commented on content/blog/software-metrics-workshop-report.md:
```diff
@@ -0,0 +1,284 @@
+---
+
+
+The mission of the URSSI conceptualization is to understand the challenges that researchers face regarding sustainable research software. One particular challenge identified in our first workshop was the lack of consensus on credit for research software. There is a need to gain a deeper understanding of the concept of credit for research software and how, as a community, we could define and measu ...
```
AFTER
The mission of the URSSI conceptualization is to understand the challenges that researchers face regarding sustainable research software
ADD: and to design a potential institute that would address these challenges.

danielskatz replied:
"topic is greater" -> "topic in greater"

danielskatz commented on content/blog/software-metrics-workshop-report.md:
```diff
@@ -0,0 +1,284 @@
+
+**Discussion topics**
+
+We began the workshop with a group discussion on whether we should focus on software credit and incentives in general or focus specifically on machine actionable/automated ways of assessing software leading which lead to credit measures and incentives. 
```
"assessing software leading which lead" -> "assessing software that lead"

danielskatz commented on content/blog/software-metrics-workshop-report.md:
```diff
@@ -0,0 +1,284 @@
+
+*   New and experienced package authors
+*   Those that rely on scripts as part of their workflow.
+*   How can researchers who share code (that is not packaged in software) get credit for their work in a way that is analogous to credit for methodology?
```
this and the two next items don't really fit the "Such an institute should support:" header - maybe reword?

danielskatz commented on content/blog/software-metrics-workshop-report.md:
```diff
@@ -0,0 +1,284 @@
+*   Can researchers get credit for lowering the cost of understanding and extending research (through software)?
+*   The institute could work towards helping scientists get to the next stage from simply sharing scripts. This could take the form of training and support to develop vetted, published software packages that follow community centric best practices. 
+
+    The institute could also help create metrics that would appeal to tenure committees. Software is making its way to[ tenure letters](https://cos.io/blog/are-reproducibility-and-open-science-starting-matter-tenure-and-promotion-review/) and URSSI could play a better role in placing research software in the right context for evaluation. \
```
"would appeal to" -> "would be useful for"
"[ tenure letters]" -> " [tenure letters]"
"play a better role" -> "play a role"

danielskatz commented on content/blog/software-metrics-workshop-report.md:
```diff
@@ -0,0 +1,284 @@
+
+
+
+**Lightning talks**
```
The spacing in this section is weird - maybe make all of these elements of a bulleted list?

danielskatz commented on content/blog/software-metrics-workshop-report.md:
```diff
@@ -0,0 +1,284 @@
+[Recommendations on Software and Service Citations from ESIP](https://github.com/si2-urssi/software-credit/blob/master/lightning-talks/jhausman_softservesip_v2.pdf) - Jessica Hausman \
+[CiteAs.org: Discovering and Improving software requests for citation](http://urssi.us/blog/2018/10/01/citeas.org-discovering-and-improving-software-requests-for-citation/) - James Howison
+
+ \
```
remove this \

danielskatz commented on content/blog/software-metrics-workshop-report.md:
```diff
@@ -0,0 +1,284 @@
+
+We focused the discussion on what an ideal credit scenario might look like and how to better reward software through citation. In particular we tried to unpack the intent of citation in the context of research software:
+
+- Do I want to cite software to **provide credit**
```
maybe add "?" to the end of this and the other 3 items?

danielskatz commented on content/blog/software-metrics-workshop-report.md:
```diff
@@ -0,0 +1,284 @@
+
+3.	A set of **coherent actions** that are designed to carry out the guiding policy. These are steps that are coordinated with one another to work together in accomplishing the guiding policy.
+
+First, the attendees, working with the overall challenge of making research software more sustainable, determined a set of eight challenges, six of which are related to credit: \
```
remove trailing \

danielskatz commented on content/blog/software-metrics-workshop-report.md:
```diff
@@ -0,0 +1,284 @@
+
+**6.	There is a lack of [funding opportunities | stable funding] for maintenance of software that is important but doesn't have a generic market**
+
+**And two of which are not:**
```
remove bolding

danielskatz commented on content/blog/software-metrics-workshop-report.md:
```diff
@@ -0,0 +1,284 @@
+
+Second, the attendees developed a set of guiding policies for an institute, which the groups modified during the breakouts sessions to:
+
+•	Leverage existing organizations for authority, credibility, resources
```
change list to normal bullets/items?

danielskatz commented on content/blog/software-metrics-workshop-report.md:
```diff
@@ -0,0 +1,284 @@
+
+While the detailed results follow, the important points that came out of this activity are:
+
+•	In general, actions support more than one challenge. In other words, the sets of actions for each challenge have overlaps with actions from other challenges.
```
change list to normal bullets/items?

danielskatz commented on content/blog/software-metrics-workshop-report.md:
```diff
@@ -0,0 +1,284 @@
+
+•	In general, actions support more than one challenge. In other words, the sets of actions for each challenge have overlaps with actions from other challenges.
+
+•	Related to this, while the cost of the set of actions for each challenge if simply added across all the challenges would be about $2.7m-$3m/year which far exceeds the $1m/year budget, the overlaps offer the promise to reduce this cost
```
change list to normal bullets/items?

danielskatz replied:
and do this for all remaining lists

danielskatz commented:
I've added a bunch of comments, mostly very minor.

I also wonder if we should briefly summarize the overall results in the first paragraph, and then let the rest be a more detailed summary?

karthik commented:
> I've added a bunch of comments, mostly very minor.

I fixed most of these. 

> I also wonder if we should briefly summarize the overall results in the first paragraph, and then let the rest be a more detailed summary?

I'll work on a short paragraph providing a summary

karthik commented:
@danielskatz I added a summary. Can you take another quick look at the post?

👉 [Preview of post](http://urssi.us/blog/2019/03/24/report-from-urssi-workshop-on-software-credit-citation-and-metrics/)

danielskatz commented:
lgtm
